python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py

```diff
@@ -0,0 +1,649 @@
+        super().validate()
+        req_params = (self.nbins, self.bin_size, self.min_sep, self.max_sep)
+        num_req_params = sum(param is not None for param in req_params)
+        if num_req_params != 3:
+            msg = (
+                "You must specify exactly three of ``nbins``, ``bin_size``, ``min_sep`` and ``max_sep``"
+                f" in treecorr_config. {num_req_params} parameters were set instead."
+            )
+            raise FieldValidationError(self.__class__.bin_size, self, msg)
+
+        if self.min_sep is not None and self.max_sep is not None:
+            if self.min_sep > self.max_sep:
+                raise FieldValidationError(self.__class__.min_sep, self, "min_sep must be <= max_sep")
+
+        if self.min_rpar is not None and self.max_rpar is not None:
+            if self.min_rpar > self.max_rpar:
+                raise FieldValidationError(self.__class__.min_rpar, self, "min_rpar must be <= max_rpar")
+
+    def toDict(self):
+        # Docs inherited from base class
+        # We are excluding items that are None due to treecorr limitations.
+        # TODO: DM-38480. This override can be removed after treecorr v4.3
+        # makes its way onto rubin-env.
+        return {k: v for k, v in super().toDict().items() if v is not None}
+
+
+class CalcRhoStatistics(KeyedDataAction):
+    r"""Calculate rho statistics
+
+    Rho statistics refer to a collection of correlation functions involving
+    PSF ellipticity and size residuals. They quantify the contribution from PSF
+    leakage due to errors in PSF modeling to the weak lensing shear correlation
+    functions. The standard rho statistics are indexed from 1 to 5, and
+    this action calculates a sixth rho statistic, indexed 0.
+
+    Notes
+    -----
+    The exact definitions of rho statistics as defined in [1]_ are given below.
+    In addition to these five, we also compute the auto-correlation function of
+    the fractional size residuals and call it as the :math:`\rho_0( \theta )`.
+
+    .. math::
+
+       \rho_0(\theta) &= \left\langle \frac{\delta T_{PSF}}{T_{PSF}}(x) \frac{\delta T_{PSF}}{T_{PSF}}(x+\theta) \right\rangle  # noqa: E501, W505
+
+       \rho_1(\theta) &= \langle \delta e^*_{PSF}(x) \delta e_{PSF}(x+\theta) \rangle  # noqa: W505
+
+       \rho_2(\theta) &= \langle e^*_{PSF}(x) \delta e_{PSF}(x+\theta) \rangle
+
+       \rho_3(\theta) &= \left\langle (e^*_{PSF}\frac{\delta T_{PSF}}{T_{PSF}}(x)) \delta e_{PSF}(x+\theta) \right\rangle  # noqa: E501, W505
+
+       \rho_4(\theta) &= \left\langle (\delta e^*_{PSF}(x) (e_{PSF}\frac{\delta T_{PSF}}{T_{PSF}}(x+\theta)) \right\rangle  # noqa: E501, W505
+
+       \rho_5(\theta) &= \left\langle (e^*_{PSF}(x) (e_{PSF}\frac{\delta T_{PSF}}{T_{PSF}}(x+\theta)) \right\rangle  # noqa: E501, W505
+
+    The definition of ellipticity used in [1]_ correspond to ``epsilon``-type ellipticity, which is typically
+    smaller by a factor of 4 than using ``chi``-type ellipticity.
```

Comment:
epsilon and chi are not standard names for what you're referring to here.  The more typical names are shear vs distortion.  epsilon is sometimes used to refer to shear, and chi is sometimes distortion.  But not by any stretch universally.  So you should at least give the names in addition to (or instead of) the Greek letters.